rosa_planning/rosa-sts-ocm-role.adoc

```diff
@@ -6,47 +6,29 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 
 toc::[]
 
-{product-title} (ROSA) web UI requires that you have specific permissions on your AWS account that create a trust relationship to provide the end-user experience at {cluster-manager-url} and for the `rosa` command line interface (CLI).
+You must create a number of role resources on your AWS account in order to create and manage a {product-title} (ROSA) cluster.
```

Comment:
AFAICT, this file is excluded from the preview (via "#. . ." in the topic map). The content looks good, but I'm unable to see how it renders, unless I'm missing something.